examples/kokkos-tutorials/workload/bfs_bottomup.py

```diff
@@ -0,0 +1,170 @@
+from typing import Tuple
+
+import pykokkos as pk
+
+from parse_args import parse_args
+
+@pk.workload
+class Workload:
+    def __init__(self, N: int, M: int):
+        self.N: int = N
+        self.M: int = M
+
+        self.val: pk.View1D[pk.double]       = pk.View([N*M], pk.double)
+        self.visited: pk.View1D[int]         = pk.View([N*M], int)
+        self.mat: pk.View2D[pk.double]       = pk.View([N, M], pk.double)
+        self.max_arr: pk.View1D[pk.double]   = pk.View([N], pk.double)
+        self.max_arr2D: pk.View2D[pk.double] = pk.View([N, N], pk.double)
```

Comment:
Try to follow the formatting in the other examples, no need for this many spaces